components/MealList.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+import { Vibration } from 'react-native'
+import styled from 'styled-components/native'
+
+import backgroundImage from '../assets/background-fruits.png'
+
+const MealsArray = [
+  {
+    meals: 'GREEK SALAD',
+    picture: require('../assets/grekisksallad.jpeg')
+  },
+  {
+    meals: 'CLUB SANDWICH',
+    picture: require('../assets/clubsandwich.jpeg')
+  },
+  {
+    meals: 'CHICKEN STEW',
+    picture: require('../assets/kycklinggryta.jpeg')
+  },
+  {
+    meals: 'CHICKEN PASTA',
+    picture: require('../assets/kycklingpasta.jpeg')
+  },
+  {
+    meals: 'BOLOGNESE',
+    picture: require('../assets/kottfarssas.jpeg')
+  },
+  {
+    meals: 'LASAGNE',
+    picture: require('../assets/lasagne.jpeg')
+  },
+  {
+    meals: 'SALMON SALAD',
+    picture: require('../assets/laxsallad.jpeg')
+  },
+  {
+    meals: 'PASTA PESTO',
+    picture: require('../assets/pastapesto.jpeg')
+  },
+  {
+    meals: 'STEAK STEW',
+    picture: require('../assets/steakstew.jpeg')
+  },
+  {
+    meals: 'PANCAKES',
+    picture: require('../assets/pannkakor.jpeg')
+  },
+  {
+    meals: 'PIZZA',
+    picture: require('../assets/pizza.jpeg')
+  },
+  {
+    meals: 'QUESADILLA',
+    picture: require('../assets/quesadilla.jpeg')
+  },
+  {
+    meals: 'HAMBURGER',
+    picture: require('../assets/hamburgare.jpeg')
+  }
+
+]
+
+const MealList = () => {
+  const [meal, setMeals] = useState({})
+
+  const setRandomMeals = () => {
+    const allTheMeals =
+      MealsArray[Math.floor(Math.random() * MealsArray.length)];
+    setMeals(allTheMeals);
+  }
+
+  return (
+    <Container>
+      <BackgroundImageContainer
+        source={backgroundImage}>
+        <ButtonContainer
+          onPress={() => {
+            setRandomMeals()
+            Vibration.vibrate()
```

Comment:
If we need to pass more than one line of code to event handler, then we should move whole logic to separate function

Reply:
I think I changed it right now - please check line 69 + 70 and 78 in the component MealList :)